test/test_distributions.py

```diff
@@ -75,11 +85,40 @@ def test_normal(self):
         self.assertEqual(Normal(mean_1d, std_1d).sample().size(), (1,))
         self.assertEqual(Normal(0.2, .6).sample_n(1).size(), (1, 1))
         self.assertEqual(Normal(-0.7, 50.0).sample_n(1).size(), (1, 1))
+        
```

Comment:
nit: We should eventually split these tests up into smaller pieces, but that can wait.